src/java/fr/paris/lutece/portal/service/mail/MailAttachmentCacheService.java

```diff
@@ -71,7 +71,7 @@ public static MailAttachmentCacheService getInstance( )
     {
         if ( _singleton == null )
         {
-            synchronized( MailAttachmentCacheService.class )
+            synchronized ( MailAttachmentCacheService.class )

```

Comment:
Same remark here for the synchronisation scope

Reply:
Corrected in another commit of the same PR.